adam/curriculum/phase1_curriculum.py

```diff
@@ -582,6 +583,38 @@ def _make_roll_curriculum():
     )
 
 
+def _make_drink_curriculum():
```

Comment:
Can you add a test for the language generation, which ensures that the container is not mentioned? (e.g. `a baby drinks juice`)